src/core/issues-notes.js

```diff
@@ -311,4 +311,8 @@ export async function run(conf) {
     ednote.classList.remove("ednote");
     ednote.classList.add("note");
   });
+  const notes = document.querySelectorAll(".note");
```

Comment:
This definitely get the job done, but it's rather inefficient, because we already did the query for `.note` a few lines above (line 295). 

I was expecting `handleIssues()` around the following line to add this attribute:

https://github.com/w3c/respec/blob/c98b2a7d76cb77a0c077a6619dd7c810eb720269/src/core/issues-notes.js#L53

But only if the `type === "note"`.